src/glcontext_egl.cpp

```diff
@@ -212,11 +221,15 @@ EGL_IMPORT
             uint32_t msaaSamples = msaa == 0 ? 0 : 1<<msaa;
 			m_msaaContext = true;
 #endif // BX_PLATFORM_ANDROID
+           
+		    bool headless = NULL == nwh;
 
 			EGLint attrs[] =
 			{
 				EGL_RENDERABLE_TYPE, (gles >= 30) ? EGL_OPENGL_ES3_BIT_KHR : EGL_OPENGL_ES2_BIT,
 
+				EGL_SURFACE_TYPE, headless ? EGL_PBUFFER_BIT : EGL_WINDOW_BIT,
```

Comment:
If it possible to be surfaceless in headless mode?

Reply:
Yes you can be surfaceless in headless or non headless, if the local platform exposes the functionality. (but see my comment above)